### ISSUE TITLE
Is there a way to have the full commit message included in the changelog?

### ISSUE THREAD

**Issue by NullVoxPopuli** [semantic-release/changelog#142]
It looks like at the moment release notes are just the commit subject.
Is there a way to use the body as well?

When merging PRs, I'd like to fill out a bunch of details about the change, impact, etc
My current config is just:
```
"release": {
    "branches": [
      "main",
      "master"
    ],
    "plugins": [
      "@semantic-release/commit-analyzer",
      "@semantic-release/release-notes-generator",
      "@semantic-release/changelog",
      "@semantic-release/npm",
      "@semantic-release/github",
      "@semantic-release/git"
    ]
  },

```

**Comment by trallnag** [comment]
Yeah would be nice. Currently the only way to do this is editing the changelog and release after running semantic release.

**Comment by trallnag** [comment]
Question is how the full commit message should be formatted in the changelog. Also every item in the changelog contains a link to the specific commit. So at least with using platforms like GitHub or GitLab you can easily look up the respective commit including the full commit message.

![image](https://user-images.githubusercontent.com/24834206/132094527-ba47d058-336e-4474-8fc6-38e7199501e5.png)

![image](https://user-images.githubusercontent.com/24834206/132094535-5ba8c867-7efc-4dd6-8727-0abde764857d.png)
